projects/graph/src/graph.py

```diff
@@ -6,4 +6,20 @@
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
-        pass  # TODO
+        
+        self.vertices = {}
+    def add_vertex(vertex_id):
+        self.vertices[vertex_id] = Vertex(vertex_id)
+    def add_edge(self, v1, v2):
+        if v1 in self.vertices and v2 in self.vertices:
+            self.vertices[v1].edges.add(v2)
+            self.vertices[v2].edges.add(v1)
+        else: 
+            reaise IndexError("That vertex does not exist")
+
+class Vertex:
+    def __init__(self, vertex_id):
+        self.id = vertex_id
+        self.edges = set()
+    def __repr__(self):
+        return (f'{self.edges}')
```

Comment:
I like your use of the fstring here. Now that you have the Vertex class up and running, things should go much faster and smoother from here on.